### PR TITLE
[teleop] Add first waypoint lat/lon to logging

### PR DESCRIPTION
**Pull request by CameronTressler**
Log data from the first waypoint in the current course.

### CODE REVIEW

File: base_station/gui/src/components/WaypointEditor.vue [hunk 1]
```diff
@@ -133,37 +133,37 @@ export default {
...
-                      bearing_deg: 0,
-                      speed: -1,
-                      type: "Odometry"
-                  },
-              }
-            })
-        };
-        course.hash = fnvPlus.fast1a52(JSON.stringify(course));
-        course.type = 'Course'
-        this.$parent.publish('/course', course)
```
Comment (CameronTressler) [inline base_station/gui/src/components/WaypointEditor.vue:166]: No actual changes here, formatting was just bugging me and it's somewhat related

**Review by cgiger00 — APPROVED** [review]
lgtm

**Review by zwgold — APPROVED** [review]
p solid, lgtm

**Review by rbridges12 — APPROVED** [review]
looks good

**Review by niwhsa9 — APPROVED** [review]
lgtm

**Pull request merged**